src/onegov/org/models/page.py

```diff
@@ -84,6 +84,26 @@ def allowed_subtraits(self):
 
         raise NotImplementedError
 
+    @property
+    def show_western_name_order(self):
+        """
+        Shall people names of current topic be shown in western order?
+        Default is False
+
+        """
+        return self.content.get('western_name_order', False)
+
+    @show_western_name_order.setter
+    def show_western_name_order(self, value):
+        """
+        Sets whether people of current topic shall be shown in western
+        order.
+
+        Example: "John Doe" (western) instead of "Doe John" (eastern)
+
+        """
+        self.content['western_name_order'] = value
+
```

Comment:
Why not add this as part of the extension? Also why not use `western_name_order = content_property(default=False)`?